tests/conftest.py

```diff
@@ -13,34 +13,82 @@
 #  permissions and limitations under the License.
 import logging
 import os
+import shutil
+import sys
 
 import pytest
 
 from zenml.artifacts.base_artifact import BaseArtifact
 from zenml.constants import ENV_ZENML_DEBUG
 from zenml.core.repo import Repository
-from zenml.exceptions import InitializationException
 from zenml.materializers.base_materializer import BaseMaterializer
 from zenml.pipelines import pipeline
 from zenml.steps import StepContext, step
 
 
-def pytest_sessionstart(session):
-    """Called after the Session object has been created and
-    before performing collection and entering the run test loop.
-    """
+@pytest.fixture(scope="session", autouse=True)
+def base_repo(tmp_path_factory, session_mocker):
```

Comment:
So this is getting cached and not being run on every test right?

Reply:
Yes